backend/docs/docs.yaml

```diff
@@ -664,9 +660,12 @@ components:
         title:
           type: string
           example: Epidemia
-        imdbNote:
+        imdbCode:
```

Comment:
It should be a IMDb note (probably short description about the movie), not IMDb code.
From subject “ In addition to the name of the video, a thumbnail must be composed, if available, of its production year, its IMDb note and a cover image.”

Reply:
I tried looking for the French subject to translate the original thing by myself because _note_ is such a confusing word as it could also be the **rating**. (Rating is _notation_ in French). The IMDB code is included there as it is the 'primary key' to GET the single movie using /api/movies/:id route.

I can change the endpoint to include the description of the film if we think that is the right use case here. Length of the said description is usually around 100-200 characters

Reply:
IMDb rating for move card makes sense. We have a sort by rating on that page, it could help user to judge correctness of the selection

Reply:
Do you all agree on my interpretation of _IMDb note_ meaning IMDb **rating** instead of a short description @DianaMukaliyeva  @iljaSL @lapaset @T7Q ?

If so, I'll just keep the response object properties like in the commit with `imdbRating` and `imdbCode`. It should be easy to modify them later if need be (and to add the film description if we want it in the movie cards).